subject_files_status_for_dpdash2.py

```diff
@@ -28,8 +28,22 @@ def get_value(event,var):
     return ''
 
 
+def get_value_from_dict(dict1, event, var):
```

Comment:
This proposed function should be removed. This is a clone of the original `get_value(event,var)` function in line 16 with no changes in body. Tashrif has designed it in a way that `dict1` is already available within the function.

Reply:
By this logic, all changes from `get_value` to `get_value_from_dict` are void.

Reply:
Using a non-defined variable inside a function makes unit testing difficult, but if the original `get_value` works as expected without taking `dict1` as input, I'm more than happy to remove `get_value_from_dict`.

Reply:
My understanding was that as long as variables are defined in main scope, they become available in spawned functions. Regardless, I can take care of the removal and co-author you.

Reply:
> Using a non-defined variable inside a function makes unit testing difficult

I acknowledge it. Please see:
https://github.com/AMP-SCZ/utility/pull/106#issuecomment-1954968985